sechub-administration/src/test/java/com/daimler/sechub/domain/administration/project/ProjectAdministrationRestControllerMockTest.java

```diff
@@ -162,6 +176,37 @@ public void delete_project_calls_delete_service() throws Exception {
         verify(projectDeleteService).deleteProject("projectId1");
     }
 
+    @Test
+    public void change_project_calls_change_details() throws Exception {
+
+        /* execute + test @formatter:off */
+        this.mockMvc.perform(
+                put(https(PORT_USED).buildAdminChangesProjectUrl(PROJECT_ID.pathElement()), "project1").
+                contentType(MediaType.APPLICATION_JSON_VALUE).
+                accept(MediaType.APPLICATION_JSON_VALUE).
+                content("{\"description\":\"new description\"}")
+                ).
+        andExpect(status().isOk());
+                
+        verify(detailChangeService).changeDetails(matches("project1"), any());
+        /* @formatter:on */
+    }
+    
```

Comment:
Can you add tests for what happens if you issue a `PUT` request in the following situations:

1. the project does not exist.
2. no changes to the project.
3. something other than the description changed.

Reply:
As we discussed, I added this tests to a `ProjectDetailChangeServiceTest`